modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/src/main/resources/META-INF/resources/js/hooks/useOnClickOutside.es.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import dom from 'metal-dom';
```

Comment:
@alinedoleron, do we need to use Metal for this purpose? Isn't anything similar on React (maybe `react-dom`?) or JavaScript vanilla?

Reply:
I'll check if it has the same use on react-dom. I used this code which Bui had sent to me. So I copied the same code, but I didn't realize it used metal.